src/proxy/socks5.rs

```diff
@@ -15,13 +15,15 @@ pub struct Socks5 {
     cfg: Config,
     cert_manager: identity::SecretManager,
     workloads: WorkloadInformation,
+    hbone_port: u16,
```

Comment:
should this go on `Config` to avoid changing so many signatures?

Reply:
The issue is we don't know the port until after `inbound` starts

Reply:
One way we could do it is make it an Option and then swap it in later, but its a bit wonky...

Reply:
yeah misunderstood at first